.travis.yml

```diff
@@ -10,6 +10,7 @@ php:
   - 5.6
   - 7.0
   - 7.1
+  - 8.0
```

Comment:
Add 7.2, 7,3 and 7.4 as well

Reply:
Looks like tests on PHP 8 fail. Investigate what's wrong and fix it please.

Reply:
Travis doesn't seem to want to pick up the build anymore

Reply:
Interesting. Sadly I have no idea why.

Reply:
Nevermind its picked it up now

Reply:
Could you add php 8.1?
And there seems to be a notice/warning since php 7.4.

Reply:
Can't really now as travis.org has ceased so the CI needs rewriting (I suggest to rewrite into GitHub Actions)